docs/getting_started.md

```diff
@@ -157,6 +169,45 @@ quay:
   access_token: 'YOUR_TOKEN'
 ```
 
+## Deploying Clusters and Running Tests
```

Comment:
We have examples of those kind of usages you are trying to describe here in this (and other below) section(s) in different MD file for `usage` which content you can see here: https://ocs-ci.readthedocs.io/en/latest/docs/usage.html .

Have you seen this file? And hope you've seen whole generated documentation on readthedocs.io :)

So if you miss things there you can add it to this file and rather link this file here in getting started to continue with usage and link user directly to different documentation file for usage.

Reply:
I have tried to read every document linked from the initial readme.
It's my strong belief that newcomers should not have to hop around from document to document to get an introduction to a new project. The getting started document should cover all the basics needed to, well, "get started" with the project.

Reply:
The getting started is mainly covering how to install and all the requirements needed and so on, then the usage is described in the usage.md file which we should probably linked from this getting started md file.   I think that if user will see the link all the examples about usage can live in separate md file .  Then we can have only one MD file and any structure which I think is better to have some separation.

Reply:
I can't really agree to that. Getting up to speed on a new project puts a fair amount of mental load on a person even if they're fairly experienced. Giving a person a "one stop shop" to get an introduction and good feel for the project before introducing complications is a much smoother path IMO. Another document can and should be used to add more complexities and other use cases but the newcomer shoudn't need to have N documents open just to learn the basics.

Reply:
I wanted to avoid to have duplications where we will cover the same on multiple places as I know we cover this already in usage md file.  I've added this topic to our mtg agenda for automation syncup which starting in about 1 hour 40 mins and it's in OCS calendar.  I will ask other people to review this change to have more opinions here.  Thanks

Reply:
I think it would be fair to have a few *basic* usage examples in our `getting started` document while having more complex / involved usage examples in the standalone examples doc. While the separation of getting started (prereqs and installation) and our usage doc seems to make sense to us, we all have experience with the framework so we really aren't the target audience. I agree that ramping up on a project can be daunting and having a "one stop shop" for users to go to in order to get up to speed will help us and them out in the long run.

My vote is to have a few basic usage examples in the getting started guide while still maintaining a full-fledged usage doc to house the usage examples for more complex scenarios.

Reply:
We can add 1 or 2  usage example but please also link the usage.md we already have which will be more detailed with all cases.